charts/guestbook/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
@duglin sure, so as I understood, value.yaml is mainly for values that chart author wants to be configurable by users. Also, since helpers are important features of helm I wanted to give an example of it as part of this chart. Hope makes sense. Thanks!